Sources/PointFree/Blog/BlogAtomFeed.swift

```diff
@@ -28,6 +28,6 @@ private func atomEntry(for post: BlogPost) -> AtomEntry {
     title: post.title,
     siteUrl: url(to: .blog(.show(post))),
     updated: post.publishedAt,
-    content: [text(post.blurb)]
+    content: blogPostContentView.view(post)
```

Comment:
just wanna double check that you are cool with rendering the full blog post content to the atom feed.

Reply:
yup!